docs/release-train-issue-template.md

```diff
@@ -0,0 +1,69 @@
+Release Akka Projections $VERSION$
+
+<!--
+
+(Liberally copied and adopted from Scala itself https://github.com/scala/scala-dev/blob/b11cd2e4a4431de7867db6b39362bea8fa6650e7/notes/releases/template.md)
+
+For every release, make a copy of this file named after the release, and expand the variables.
+Ideally replacing variables could become a script you can run on your local machine.
+
+Variables to be expanded in this template:
+- $VERSION$=??? 
+
+-->
+
+### before the release
+
+- [ ] Create a new milestone for the [next version](https://github.com/akka/akka-projection/milestones)
+- [ ] Check [closed issues without a milestone](https://github.com/akka/akka-projection/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20no%3Amilestone) and either assign them the 'upcoming' release milestone or `invalid/not release-bound`
+- [ ] Make sure all important / big PRs have been merged by now
+
+### Preparing release notes in the documentation / announcement
+
+- [ ] For non-patch (minor/major) releases: rename the 'akka-projection-x.x-stable' and 'akka-projection-supported-x.x-stable' reporting projects in [WhiteSource](https://saas.whitesourcesoftware.com/Wss/WSS.html) accordingly (unfortunately this requires permissions that cannot be shared outside of Lightbend)
+- [ ] For non-patch releases: Create a news item draft PR on [akka.github.com](https://github.com/akka/akka.github.com), using the milestone
+- [ ] Move all [unclosed issues](https://github.com/akka/akka-projection/issues?q=is%3Aopen+is%3Aissue+milestone%3A$VERSION$) for this milestone to the next milestone
+- [ ] Close the [milestone](https://github.com/akka/akka-projection/milestones?direction=asc&sort=due_date)
+
+### Cutting the release
+
+- [ ] Make sure any running [Travis builds](https://travis-ci.com/akka/akka-projection) for the commit you would like to release have completed.
```

Comment:
I changed the travis links to ci.com
why do we have some projects on ci.com and some on ci.org? historical reasons?